dist-mcp/README.md

```diff
@@ -12,7 +12,7 @@ To use this MCP server with Cursor, add the following to your Cursor config:
   "foundry": {
     "command": "bun",
     "args": [
-      "/Users/bskvorc/repos/k-start/dist-mcp/index.js"
+      "/Users/williamkempster/projects/specialk/dist-mcp/index.js"
```

Comment:
Let's undo this change - this should actually be ignored or have a generic non-user-specific route.

Reply:
done. imagine you're aware but the change to the readme is caused by this line when running `build:all` - https://github.com/kempsterrrr/specialk/blob/e977691d674febe7c2603fd878ce8c987c4ade2a/build.js#L108